src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -292,12 +292,27 @@ private static string CreateArgumentList(IEnumerable<string> args)
                     if (stringBuilder.Length != 0)
                         stringBuilder.Append(' ');
 
-                    stringBuilder.Append(Terminal.Quote(arg));
+                    stringBuilder.Append(QuoteArgument(arg));
                 }
             }
             return stringBuilder.ToString();
         }
 
+        // gdb does not like parenthesis without being quoted
+        private static char[] s_CHARS_TO_QUOTE = { ' ', '\t', '(', ')' };
+        private static string QuoteArgument(string argument)
+        {
+            // Make sure its not empty or already quoted (starts with a quote)
+            if (!string.IsNullOrWhiteSpace(argument) && argument[0] != '"')
```

Comment:
If an argument is something like: `"args": [ "\"Hello World\"" ]`

Shouldn't we really be giving GDB some some sort of arg that has escaped quotes, rather than raw quotes so that in the C program args[1] is `"\"Hello World\""` instead of `"Hello World"`?